### PR TITLE
Resolve the working of multi column layout in favourites page

### PR DESCRIPTION
**Pull request by Palakkgoyal**
This PR resolves issue no. #479
Now, the layout is working fine just same as the home page.

### CODE REVIEW

**Review by narayan954 — APPROVED** [review]
looks good to me!

File: src/App.jsx [hunk 1]
```diff
@@ -185,163 +186,165 @@ function App() {
...
-              src="https://user-images.githubusercontent.com/27727921/185767526-a002a17d-c12e-4a6a-82a4-dd1a13a5ecda.png"
-              alt="dummygram"
-              className="modal__signup__img"
-              style={{
-                width: "80%",
-                marginLeft: "10%",
-                filter: "invert(var(--val))",
-              }}
-            />
+    <RowModeContext.Provider value={rowMode}>
```
Comment (narayan954) [inline src/App.jsx:189]: instead of defining the context at app, can you define it at index/main where other contexts are defined as well?

**Review by narayan954 — CHANGES_REQUESTED** [review]
small change

File: src/App.jsx (hunk 1 above)
Comment (narayan954) [reply src/App.jsx:189]: that way we won't have to pass it as a prop to navbar as well

Comment (Palakkgoyal) [reply src/App.jsx:189]: So, you are saying that I should wrap the app with context provider in main file like in the screenshot and update it in navbar. Right?

Comment (narayan954) [reply src/App.jsx:189]: yess

**Comment by Palakkgoyal** [comment]
Hey @narayan954 according to me we should add context provider to main otherwise we would also need to pass state i.e. rowMode and setRowMode to main and then, the main file will also become cluttered like App.jsx 
I have done some changes and used context in Navbar. Please review them.

**Comment by narayan954** [comment]
> Hey @narayan954 according to me we should add context provider to main otherwise we would also need to pass state i.e. rowMode and setRowMode to main and then, the main file will also become cluttered like App.jsx 
> I have done some changes and used context in Navbar. Please review them.

yeah ,seems right! will review and merge asap

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Pull request merged**